anyway/infographics_utils.py

```diff
@@ -252,6 +252,89 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
 _("Fatal, severe and light injured count in the specified years, split by injury severity")
 _("Fatal, severe and light accidents count in the specified years, split by accident severity")
 
+@register
+class SevereFatalCountByAccidentYearAndVehicleWidget(UrbanWidget):
+    name: str = "severe_fatal_count_by_vehicle_by_accident_year"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 15
+        self.text = {
+            "title": "Severe or fatal accidents on bikes, e-bikes, or scooters in "
+            + self.request_params.location_info["yishuv_name"]
+        }
+
+    def generate_items(self) -> None:
+        self.items = {"e_bikes": get_accidents_stats(
+            table_obj=InvolvedMarkerView,
+            filters = {
+                "injury_severity": [1, 2],
+                "involve_vehicle_type": 23,
+                "involve_yishuv_name": self.request_params.location_info["yishuv_name"],
+                },
+            group_by="accident_year",
+            count="accident_year",
+            start_time=self.request_params.start_time,
+            end_time=self.request_params.end_time,
+        ),
+        "bikes": get_accidents_stats(
+            table_obj=InvolvedMarkerView,
+            filters = {
+                "injury_severity": [1, 2],
```

Comment:
Please use constants from backend_constants for values of types like injury_severity and the like.